server/routes/user.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const multer = require('multer');
+const { GridFsStorage } = require('multer-gridfs-storage');
+const Grid = require('gridfs-stream');
+
+// Setup MongoDB connection
+const mongoURI = process.env.MONGO_URI;
+const conn = mongoose.createConnection(mongoURI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+});
+
+// Initialize GridFS
+let gfs;
+conn.once('open', () => {
+    gfs = Grid(conn.db, mongoose.mongo);
+    gfs.collection('avatars');
+});
+
+// Storage engine for multer and GridFS
+const storage = new GridFsStorage({
+    url: mongoURI,
+    file: (req, file) => {
+        return {
+            filename: `${req.body.username}-avatar-${Date.now()}`,
+            bucketName: 'avatars',
+        };
+    },
+});
+const upload = multer({ storage });
+
+// Define user schema with bio field
+const userSchema = new mongoose.Schema({
```

Comment:
We have a user schema in `models/user.js` with more details that you can import

Reply:
got it